### DISCUSSION TITLE
Vertical tab filter not like operator

### DISCUSSION THREAD

**Discussion by vizsnyai** [microsoft-search/pnp-modern-search#4128] [Q&A]
Hi,
I use a vertical tab to separate internal and external employees. There is a tab that filters for domain endings. I need another tab that filters out everything else that is not the domain specified in the previous tab.
Can I use the "not like" operator here in the tab value field? 
How to display everything that is not equal to a given string?

Thanks.

**Comment by wobba** [comment]
I think we need some more information on the actual setup, what properties contain values etc. in order to provide the best answer.

**Reply by vizsnyai** [reply]
Hi,
I would like to apply a filter where I filter on the emal ending.
For this I use the following query: {inputQueryText}* -"SPS-HideFromAddressLists":1 AND (WorkEMail:{verticals.value})

Vertical tab values
tab1: ab.com
tab2: * -ab.com
What I want to achieve is that by clicking on the first tab, all items that end with ab.com should appear.
By clicking on the second tab, everything should appear except those that end with ab.com.
The problem is that there are also values ​​that end with abcd.com, so they are also filtered out.
I need an expression that allows me to exclude users with an email address ending with exactly ab.com by clicking on the second tab.

Thanks

**Reply by wobba** [reply]
You can try to use these expressions in the *Refinement filters* section of the web part setup which takes FQL. See https://learn.microsoft.com/en-us/sharepoint/dev/general-development/fast-query-language-fql-syntax-reference#ends-with

`WorkEMail:ends-with("ab.com")` vertical1 webpart
`not(WorkEMail:ends-with("ab.com"))` vertical2 webpart
